client/python/lib/ovmsclient/tfs_compat/grpc/tensors.py

```diff
@@ -0,0 +1,93 @@
+#
+# Copyright (c) 2021 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+def make_tensor_proto(values, dtype=None, shape=None, use_tensor_content=True):
+    '''
+    Create TensorProto object from values.
+
+    Args:
+
+        values: Values to put in the TensorProto.
+            The accepted types are: python scalar, pythons list, numpy scalar and numpy ndarray.
+            Python scalars and lists are internally converted to their numpy counterparts before further processing.
+            String values are converted to bytes and placed in TensorProto string_val field. 
+
+        dtype (optional): tensor_pb2 DataType value. 
+            If not provided, the function will try to infer the data type from **values** argument.
+
+        shape (optional): The list of integers defining the shape of the tensor. 
+            If not provided, the function will try to infer the shape from **values** argument.
+
+        use_tensor_content (optional): Determines if the **values** should be placed in tensor_content field of TensorProto.  
```

Comment:
currently ovms doesn't support _val types it it will not be used. It will be also much slower so I would question if we need this option.

Reply:
Stateful models use val fields for sequence ID and control input. 
It gives user more control over created proto. In case we decide to support val fields for some reasons the solution on the client side will already be in place.